addons/project/tests/test_project_tags_filter.py

```diff
@@ -0,0 +1,60 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import tagged, HttpCase
+from odoo import Command
+from .test_project_base import TestProjectCommon
+
+
+@tagged('-at_install', 'post_install')
+class TestProjectTags(HttpCase, TestProjectCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.stage_pig = cls.env['project.project.stage'].create({
+            'name': 'pig stage'
+        })
+        cls.stage_goat = cls.env['project.project.stage'].create({
+            'name': 'goat stage'
+        })
+        cls.env['project.tags'].create([
+            {'name': 'Pink', 'project_ids': [Command.link(cls.project_pigs.id)]},
```

Comment:
```suggestion
            {'name': 'Corkscrew tailed', 'project_ids': [Command.link(cls.project_pigs.id)]},
```
Sounds more characteristic :sweat_smile: